tests/basic/upgrade_cluster_test.go

```diff
@@ -89,6 +100,19 @@ var _ = Describe("{UpgradeCluster}", func() {
 					log.Warnf("PDB validation skipped. Current Px-Operator version: [%s], minimum required: [%s]. Error: [%v].", opver, PDBValidationMinOpVersion, err)
 				}
 
+				var vQuorumError error
+				// validate volume quorum during upgrade
+				if opver.GreaterThanOrEqual(ParallelUpgradeOperatorVersion) && pxVersion.GreaterThanOrEqual(ParallelUpgradePXVersion) {
+					log.Info("Starting volume quorum validation for Portworx upgrade .......")
+					stopVolumeQuorumValidationSignal := make(chan struct{})
+					go DoVolumeQuorumValidation(stopVolumeQuorumValidationSignal, &vQuorumError)
```

Comment:
can you a validation for vQuorumError, fail the test if it has any errors.

Reply:
@lsrinivas-pure this is changed back to following code because our intention is not to fail the test when runTime status of healthy node is resync. We are trying to validate here that the node which is being upgraded has runtime state clean.

if reflect.DeepEqual(nodeStatus, opsapi.Status_STATUS_STORAGE_DOWN) {
					err = fmt.Errorf("node [%s] is in %v Runtime state ", replicaNodes[i], runTimeState)
					processError(err, errChan...)
					return
				}